mmv1/third_party/terraform/services/firestore/resource_firestore_database_update_test.go.erb

```diff
@@ -183,6 +183,7 @@ resource "google_firestore_database" "default" {
   type                    = "DATASTORE_MODE"
   location_id             = "nam5"
   delete_protection_state = "%s"
+  deletion_policy         = "DELETE"
```

Comment:
This test is currently failing with:

```
    vcr_utils.go:152: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) (len=1) {
         (string) (len=15) "deletion_policy": (string) (len=7) "ABANDON"
        }
        
        
        (map[string]string) (len=1) {
         (string) (len=15) "deletion_policy": (string) (len=6) "DELETE"
        }
```

This is expected - deletion_policy is client-side only, so it will not be set on import. You should add it to ImportStateVerifyIgnore for any test steps calling this function.